### ISSUE TITLE
import not working

### ISSUE THREAD

**Issue by jainmayur517** [AmruthPillai/Reactive-Resume#401]
I had made a resume before and also exported the JSON for future use. As of today import feature is not working at all. A black screen is displayed.

**Comment by anilgeorge04** [comment]
+1
To give more details to the issue:
- The page stops loading after the import command at `https://rxresu.me/app/builder/<uniquecode>`
- I created my JSON file using the resume-cli from jsonresume.org
---

P.S.: Thanks for building this tool, Amruth.

**Comment by hassan556** [comment]
+1
Import is not working.

**Comment by DineshSolanki** [comment]
@AmruthPillai the error happens because of the JSON schema, this imported expects a 'basics' JSON object which is not the case in those who are importing JSON and also when this app exports to resume.

**Comment by AmruthPillai** [comment]
**TL;DR: JSONResume.org imports should start working from version 2.4 onwards!**

The problem @jainmayur517 is talking about seems like he's facing issue with importing resumes that have been exported from this current version of Reactive Resume itself. I'm not too sure about that, but I just checked on the running instance and import/export from RxResume seemed to work just fine.

As for the real issue here, about JSON Resume, you are all absolutely right about that. **There's been some bad support on my end about JSON Resume and that's my mistake. I've taken some of my free time this weekend to work on it.** I basically had to just safety check for all the values we would be importing from the resume, make sure that they're there and then parse them through the app. **JSON Resume imports should work from version 2.4 onwards** 🎊 

I tested it with a few sample JSON Resumes I found on the inter-webs including this one:
https://gist.github.com/ishu3101/36186df161316ec00519389b9deec690

_If you are still not able to get it working, please do share the JSON Resume with me through email or any medium and we can get to the bottom of it._

@DineshSolanki Thank you so much for supporting by replying to other issues on the repo and also offering to help. It's not really all that complicated, the code. I made the entire app in about 2-3 days, so you can be assured that it's n00b code. Nothing fancy here :) I'd love for you to take a look at the last few commits to see what has changed, and what exactly fixed this issue. That would help I think.

Thank you so much to everyone, for your patience and support to this ambitious little project of mine :)

**Comment by DineshSolanki** [comment]
@AmruthPillai this little project of you is awesome, thank you for this. 👏
